### DISCUSSION TITLE
No module found OpenAI

### DISCUSSION THREAD

**Discussion by MJ546-hash** [iSiddharth20/Chatbot-Using-OpenAI-API#6] [Q&A] [answered]
Exception has occurred: ModuleNotFoundError
No module named 'openai'

**Comment by iSiddharth20** [comment, marked as answer]
Hello @mjerk 

#### Problem:
`ModuleNotFoundError` signifies that the required module/package/library is not installed or cannot be imported.
Additionally `No module named ...` highlights the missing module, `openai` in this case.

#### Solution:
You can install the `openai` module using the command `pip install openai` ([source](https://pypi.org/project/openai/))

#### Additional Info:
You will also need an OpenAI API Key, which you can get from [here](https://platform.openai.com/api-keys).
This key will need to be updated on `Line 3` in [this file](https://github.com/iSiddharth20/Chatbot-Using-OpenAI-API/blob/main/Code/BotDefinition.py) in the codebase for the project to function as desired.
